pkg/signals/context_test.go

```diff
@@ -1,4 +1,5 @@
-//+build !windows
+//go:build !windows
```

Comment:
Do we want to move entirely to the new directive now, or wait until `+build` is deprecated?

Reply:
I'll see if `gofmt` allows for just the new directive, my impression from reading the release notes was that it'd enforce having both styles 👀

Reply:
Didn't even get as far as running the build... GoLand immediately flags the `//go:build` with a "//go:build comment without //+build comment" error if I delete the `// +build` line